net-misc/aggregate/files/aggregate-1.6-build-fixup.patch

```diff
@@ -1,6 +1,5 @@
-diff -Nuar aggregate-1.6.orig/Makefile.in aggregate-1.6/Makefile.in
---- aggregate-1.6.orig/Makefile.in	2002-03-06 08:59:37.000000000 -0800
-+++ aggregate-1.6/Makefile.in	2008-10-10 01:31:42.613937925 -0700
+--- a/Makefile.in
++++ b/Makefile.in
```

Comment:
I guess this technically wasn't necessary, but patch cleaning for clarity is always a good thing :+1: